CMakeLists.txt

```diff
@@ -64,6 +64,9 @@ elseif (APPLE)
     set(CX_PLATFORM "darwin")
 elseif (UNIX)
     set(CX_PLATFORM "linux")
+    file(STRINGS "/etc/lsb-release" LSB_RELEASE_CONTENT)
```

Comment:
This line broke our Alpine docker build, using /etc/os-release would be [better](https://www.freedesktop.org/software/systemd/man/latest/os-release.html) as it is present in both distributions.